src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -29,8 +29,10 @@ inline static void transform_idxs_to_regs(const std::vector<size_t>& idxs, std::
     std::transform(idxs.begin(), idxs.end(), regs.begin(), [](size_t idx){return Reg64(static_cast<int>(idx));});
 }
 
-jit_container_emitter::jit_container_emitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa,
-                      const std::shared_ptr<ov::Node>& n) : jit_emitter(h, isa, n) {
+jit_container_emitter::jit_container_emitter(dnnl::impl::cpu::x64::jit_generator* h,
```

Comment:
Do you think it's a good idea to declare a bunch of aliases at the top of this file, like
`using jit_generator = dnnl::impl::cpu::x64::jit_generator`, and similar for `cpu_isa_t` and `ExpressionPtr`.
We don't use `jit_generator` from any other namespace here, so this would increase readability without any drawbacks.

Reply:
Cool point! The aliases are added